### PR TITLE
chore: change "Timeline" to "History"

### LINKED ISSUE #7667: Change ‘Timeline’ to ‘History’
Many, many users over the years are unaware of where to find Meeting History. Recently we received direct feedback that they did not expect to find meeting history under ’Timeline’.

We began to track [this request on our on own internal Problems, Ideas, and Opportunities 🔒 board](https://www.notion.so/parabol/Change-Timeline-to-History-1ce0910aa235406995fff8d34c1c7e8f). There is a lot of internal support. Let's just do it and see what happens. It's a trivial change.

### Acceptance Criteria (optional)
   - Replaced word ‘Timeline’ with ‘History’ on side navbar
   - Search and replace for other mentions in the product
   - Notified #t_growth when change has been made so that they have the opportunity to update any screenshots that they may or may not deem to be important to update

**Estimated effort:** 1 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/parabol/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by enriquesanchez**
Fixes #7667

Desktop:
213034681-5d0ed211-18fc-48ee-8938-f078d1b19b52

Mobile:
Captura de pantalla 2023-01-18 a la(s) 3 24 58 p m

## Final checklist

- [ ] I checked the [code review guidelines](../docs/codeReview.md)
- [ ] I have added [Metrics Representative](../docs/codeReview.md#metrics-representative) as reviewer(s) if my PR invovles metrics/data/analytics related changes
- [ ] I have performed a self-review of my code, the same way I'd do it for any other team member
- [ ] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [ ] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [ ] PR title is human readable and could be used in changelog

### CODE REVIEW

File: packages/client/components/Dashboard/DashSidebar.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ const DashSidebar = (props: Props) => {
         <Contents>
           <NavItemsWrap>
             <NavItem icon={'forum'} href={'/meetings'} label={'Meetings'} />
-            <NavItem icon={'timeline'} href={'/me'} label={'Timeline'} />
+            <NavItem icon={'timeline'} href={'/me'} label={'History'} />
```
Comment (enriquesanchez) [inline packages/client/components/Dashboard/DashSidebar.tsx:69]: @Dschoordsch I'm having a hard time understanding what `icon={'timeline'}` does here. I am importing the history icon a the top, but if I change it to `icon={'history'}` the icon doesn't show up. 

Would you know why?

Comment (Dschoordsch) [reply packages/client/components/Dashboard/DashSidebar.tsx:69]: Yes, `NavItem` is just a styled version of [`LeftDashNavItem`](https://github.com/ParabolInc/parabol/blob/4c8bdbe2606549847e6d9cc91ba8f29c9d538868/packages/client/components/Dashboard/DashSidebar.tsx#L43). If you look at the [`LeftDashNavItem` props](https://github.com/ParabolInc/parabol/blob/4c8bdbe2606549847e6d9cc91ba8f29c9d538868/packages/client/components/Dashboard/LeftDashNavItem.tsx#L61) you can see that `icon` is a string. So it's happening inside `LeftDashNavItem` where we map the string to an actual icon.

File: packages/client/components/Dashboard/LeftDashNavItem.tsx [hunk 2]
```diff
@@ -85,7 +77,7 @@ const LeftDashNavItem = (props: Props) => {
         {
           {
             forum: <Forum />,
-            timeline: <Timeline />,
+            timeline: <History />,
```
Comment (Dschoordsch) [inline packages/client/components/Dashboard/LeftDashNavItem.tsx:80]: You need to change the mapping here to `history: <History />,` if you want to pass `history` in `DashSidebar`.

Comment (enriquesanchez) [reply packages/client/components/Dashboard/LeftDashNavItem.tsx:80]: @Dschoordsch Ah, got it! Thank you 👍

**Comment by Dschoordsch** [comment]
@enriquesanchez it's best to re-request review next time so it appears in my list of awaiting reviews from me.

**Review by Dschoordsch — CHANGES_REQUESTED** [review]
In mobile view it still shows "Timeline"
Screenshot from 2023-01-20 17-32-24
 as the title

**Comment by enriquesanchez** [comment]
>In mobile view it still shows "Timeline" as the title

Good catch! Thank you 🙌

**Review by Dschoordsch — APPROVED** [review]

**Pull request merged**